### DISCUSSION TITLE
Git push file size limit error

### DISCUSSION THREAD

**Discussion by ToptenDev** [community/community#48170] [Repositories] [answered]
### Select Topic Area

Product Feedback

### Body

I tried to push my current repo to my global github repo. But I got the error msg like the below.
![Untitled](https://user-images.githubusercontent.com/121523915/220405577-a5392c62-1b1c-4341-8c7d-52f713134fd4.png)

**Comment by airtower-luna** [comment]
The message indicates you have files in the repository (including its history) that are larger than allowed by GitHub. See https://docs.github.com/en/repositories/working-with-files/managing-large-files/about-large-files-on-github for the limits and how to deal with larger files.

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @ToptenDev ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Comment by Juiux** [comment, marked as answer]
If you have checked all the files in your repository using git ls-files and found that none of them exceed the 100 MB limit, it's possible that the error is caused by a commit that is being pushed and contains a large file.

In this case, you can use the git rev-list command to list the commits in your local repository and their sizes. The git rev-list command shows the commit hash, the author, the date, and the commit message, as well as the size of the commit.

Reference the below one
https://www.google.com/search?q=ntcreatefile+status_object_name_not_found&rlz=1C1CHWL_enUS1038US1038&oq=ntcreatefile+status_object_name_not_found&aqs=chrome..69i57.9920j0j1&sourceid=chrome&ie=UTF-8#fpstate=ive&vld=cid:746c8fd3,vid:9FYREN6UPsM

If you find a large commit, you can use the git filter-branch command to remove the large file from the commit history. You will need to force push your changes after running this command, so make sure to communicate this to any collaborators who may be working on the repository.

Note that if you do remove a large file from your repository history, anyone who has already cloned the repository will need to re-clone it to get the updated history.
